solr/packaging/build.gradle

```diff
@@ -41,14 +40,6 @@ configurations {
   server
   docs
   docker
-  solrTgz
-  solrTgzSignature
-  solrZip
-}
-
-artifacts {
-  solrTgz(distTar)
-  solrZip(distZip)
```

Comment:
@HoustonPutman I'll try to re-instate this way of producing the tgz and zip, and then source from here in the distribution folder. I have not made up my mind whether everything should be moved to packaging folder or not...

Reply:
See https://github.com/janhoy/solr/pull/14 for an attempt that I think is about right.

Reply:
This works now